middleware/jwt.go

```diff
@@ -0,0 +1,143 @@
+package middleware
```

Comment:
echo jwt doesnt allow to change response to client (in task it should give 403 + json error if user not authorized) thats why I modify it